src/SectionPage.tsx

```diff
@@ -0,0 +1,38 @@
+import './App.css';
+import './css_reset.css'
+import './section_view.css';
+import * as types from './types';
+import {Files} from './Files';
+import {ChordProgression} from './ChordProgression';
+import {Comments} from './Comments';
+import {CreateComment} from './CreateComment';
+import {SectionTitle} from './SectionTitle';
+import {useGlobalStore} from './hooks/useGlobalStore';
+
+type SectionPageProps = {
+    projectId: string;
+    sectionId: string;
+}
+
+const SectionPage: React.FC<SectionPageProps> = ({projectId, sectionId}) => {
+    const globalStore = useGlobalStore();
+    const section = globalStore.getSection(sectionId);
+    const files = globalStore.getFilesForSection(sectionId);
+
+    const sectionPointer: types.EntityPointer = {
+        entityId: sectionId,
+        entityType: types.EntityType.SECTION,
+    };
+
+    return (
+        <div className="root">
+            <SectionTitle sectionId={sectionId} />
+            <ChordProgression chordProgression={section.chordProgression} />
+            <Files files={files} />
+            <Comments entityPointer={sectionPointer} />
+            <CreateComment entityPointer={sectionPointer} />
+        </div>
+    );
+}
```

Comment:
Here the `SectionPage` is creating an `EntityPointer` for its `SectionData` object. It then passes that to the `Comments` component, so the `Comments` component instance knows how to find the comments it needs to render. This way, the `Comments` component works with comments attached to `FileData` objects as well.

We would do this for gathering comments for a particular file:

```ts
const filePointer: types.EntityPointer = {
    entityId: fileId,
    entityType: types.EntityType.FILE,
};
```

Then render the `Comments` component like this:

```ts
<Comments entityPointer={filePointer} />
```